src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -209,9 +213,7 @@ private static bool IsDefenderEnabled()
         {
             return Process.GetProcessesByName("MsMpEng.exe").FirstOrDefault() != null;
         }
-#endif
 
-#if OS_WINDOWS
         // Returns whether or not the Windows firewall is enabled.
         private static bool IsFirewallEnabled()
```

Comment:
why `IsFirewallEnabled()` is `static`  :D

Reply:
Fixed. :) Good catch.